BackEndApp/authv1/models.py

```diff
@@ -41,9 +41,8 @@ def create(self):
             "password": hashed_password,
             "first_name": self.attributes.get("first_name", ""),
             "last_name": self.attributes.get("last_name", ""),
-            "email": self.attributes.get("email"),
+            "email": self.attributes.get("email").lower(),
             "is_verified": False,
-        }
```

Comment:
@mharyam I think removing this closing bracket for the dictionary would cause the app to break 😅